mycrt-backend/src/server.py

```diff
@@ -107,21 +108,64 @@ def get_rds_instances():
 
     return jsonify({'status': response['ResponseMetaData']['HTTPStatusCode'], 'error': response['Error']['Code']})
 
-@app.route('/login-test', methods=['PUT'])
+@login_manager.request_loader
+def load_user_from_request(request):
+    auth = request.authorization
+    if auth:
+        username = auth.username
+        password = auth.password
+        user = user_datastore.find_user(username=username)
+        if user is not None and verify_password(password, user.password):
+            return user
+    return None
+
+@app.route('/login', methods=['POST'])
 def login():
-    user_repository.register_user(username="test-user", password="test123", email='test@test.com', access_key="test_access_key", secret_key="test_secret_key")
-
+    user = current_user
+    if user is not None:
+        login_user(user)
+        return jsonify({"status" : "success",
+            "user" : {
+                "username" : user.username,
+                "password" : user.password,
+                "access_key" : user.access_key,
+                "secret_key" : user.secret_key,
+                "email" : user.email
+                }
+            }), 200
+    else:
+        return jsonify({"status" : "fail",
+            "user" : {}
+            })
 
 @login_required
 @app.route('/logout', methods=['POST'])
 def logout():
     login_manager.logout_user(current_user)
     return redirect('/', code=302)
 
+@app.route('/register', methods=['POST'])
+def register():
+    json = request.get_json()
+    username = json['username']
+    password = json['password']
+    email = json['email']
+    secret_key = json['secret_key']
+    access_key = json['access_key']
+    return jsonify({"success" : db.register_user(username, password, email, secret_key, access_key)})
+
 @app.route('/metrics', methods=['GET'])
 def get_user_metrics():
 	return jsonify(get_metrics())
 
+@app.before_first_request
+def add_test_users():
+    '''This method adds a user for testing purposes when the server starts up.
+    This happens the first time the server gets a request.
+    '''
+    db.register_user('test-user', 'password123', 'test@test.com',

```

Comment:
So these are the credentials we should use for testing login verification?

Reply:
Yea, you can add other users there if you want.
If the username and email don't exist it will make the user. If they already exist it won't.